src/main/java/com/gtnewhorizons/neid/mixins/TargetedMod.java

```diff
@@ -0,0 +1,27 @@
+package com.gtnewhorizons.neid.mixins;
+
+public enum TargetedMod {
```

Comment:
you could delete this class since you only have VANILLA

Reply:
We'll potentially need it for the meta expansion.